### PR TITLE
abort and print config.log in case of failure

### PR DESCRIPTION
**Pull request by ggouaillardet**

### CODE REVIEW

File: jenkins/open-mpi-build-script.sh [hunk 1]
```diff
@@ -166,18 +166,30 @@ echo "--> Building commit ${sha1}"
 if test -f autogen.pl; then
     echo "--> running ./autogen.pl ${AUTOGEN_ARGS}"
     ./autogen.pl ${AUTOGEN_ARGS}
+    if test $? -ne 0; then
```
Comment (bwbarrett) [inline jenkins/open-mpi-build-script.sh]: set -e is called at the top of this script; this change won't do anything.

Comment (ggouaillardet) [reply jenkins/open-mpi-build-script.sh]: i see, will fix from now

Comment (ggouaillardet) [reply jenkins/open-mpi-build-script.sh]: done !

**Comment by ggouaillardet** [comment]
@bwbarrett i made the changes and then `push --force`. i am not sure whether GitHub issued a notification or not.

File: jenkins/open-mpi-build-script.sh [hunk 2]
```diff
@@ -165,19 +165,28 @@ echo "--> Building commit ${sha1}"
 
 if test -f autogen.pl; then
     echo "--> running ./autogen.pl ${AUTOGEN_ARGS}"
-    ./autogen.pl ${AUTOGEN_ARGS}
+    if ! ./autogen.pl ${AUTOGEN_ARGS}; then
```
Comment (bwbarrett) [inline jenkins/open-mpi-build-script.sh]: What does this add?

File: jenkins/open-mpi-build-script.sh [hunk 3]
```diff
@@ -165,19 +165,28 @@ echo "--> Building commit ${sha1}"
...
+        exit 1
+    fi
 else
     if test "${AUTOGEN_ARGS}" != ""; then
 	echo "--> Being a coward and not running with special autogen arguments and autogen.sh"
 	exit 1
     else
 	echo "--> running ./atogen.sh"
-	./autogen.sh
+	if ! ./autogen.sh; then
```
Comment (bwbarrett) [inline jenkins/open-mpi-build-script.sh]: Same as above...

File: jenkins/open-mpi-build-script.sh [hunk 4]
```diff
@@ -165,19 +165,28 @@ echo "--> Building commit ${sha1}"
...
+            echo "./autogen.sh failed, ABORTING !"
+            exit 1
+        fi
     fi
 fi
 
 echo "--> running ./configure --prefix=\"${PREFIX}\" ${CONFIGURE_ARGS}"
-./configure --prefix="${PREFIX}" ${CONFIGURE_ARGS}
+if ! ./configure --prefix="${PREFIX}" ${CONFIGURE_ARGS}; then
```
Comment (bwbarrett) [inline jenkins/open-mpi-build-script.sh]: I'd combine the two commits and only change configure.

File: jenkins/open-mpi-build-script.sh (hunk 3 above)
Comment (ggouaillardet) [reply jenkins/open-mpi-build-script.sh]: Btw, I just noted a typo at line 174

File: jenkins/open-mpi-build-script.sh (hunk 4 above)
Comment (ggouaillardet) [reply jenkins/open-mpi-build-script.sh]: Will do tomorrow

**Comment by ggouaillardet** [comment]
@bwbarrett i removed the useless changes

**Comment by bwbarrett** [comment]
@ggouaillardet, apologies.   I forgot this repo was locked down and was waiting for you to merge, not waiting for the changes. Sigh...

**Pull request merged**